pkg/docker/docker.go

```diff
@@ -176,12 +177,19 @@ func (d *stiDocker) GetImageUser(name string) (string, error) {
 // IsImageOnBuild provides information about whether the Docker image has
 // OnBuild instruction recorded in the Image Config.
 func (d *stiDocker) IsImageOnBuild(name string) bool {
+	onbuild, err := d.GetOnBuild(name)
+	return err == nil && len(onbuild) > 0
+}
+
+// IsImageOnBuild provides information about whether the Docker image has
+// OnBuild instruction recorded in the Image Config.
+func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 	name = getImageName(name)
 	image, err := d.client.InspectImage(name)
 	if err != nil {
-		return false
+		return nil, err
 	}
-	return len(image.Config.OnBuild) > 0
+	return image.Config.OnBuild, nil
```

Comment:
This returns the "ONBUILD USER root" command from the original base image as "USER root" in the array?